x-pack/plugins/infra/public/components/log_stream/log_stream.tsx

```diff
@@ -223,6 +261,7 @@ Read more at https://github.com/elastic/kibana/blob/main/src/plugins/kibana_reac
       jumpToTarget={noop}
       reportVisibleInterval={handlePagination}
       reloadItems={fetchEntries}
+      onOpenLogEntryFlyout={flyoutEnabled ? handleOpenLogFlyoutClick : undefined}
```

Comment:
I'm wondering if we should write tests for this. I haven't found any tests for this component, but it could be nice to write some.